components/FeedList/FeedItem.jsx

```diff
@@ -1,19 +1,19 @@
 import PropTypes from 'prop-types';
 import {Text, View, Image, TouchableOpacity} from 'react-native';
 import styles from './style';
+import {useCallback} from 'react';
 
 const FeedItem = ({item, itemIndex, navigation}) => {
-  const diffMonths = () => {
+  const diffMonths = useCallback(() => {
     const currentDate = new Date();
     const createdDate = new Date(item.createdData);
     return (
       (currentDate.getFullYear() - createdDate.getFullYear()) * 12 +
       (currentDate.getMonth() - createdDate.getMonth())
     );
-  };
+  }, [item]);
 
   const monthsAgo = diffMonths(item);
```

Comment:
if you directly call this here, you should use useMemo here and not useCallback and function call

Reply:
Yes, of course, it`s a variable and not a function ! Thank you